integration-tests/tests/src/tests/sync/mongo-db-client.ts

```diff
@@ -29,64 +29,50 @@ type TestDocument = {
   isLast?: boolean;
 };
 
+type CollectionContext = { collection: MongoDBCollection<TestDocument> };
+type TestContext = CollectionContext & AppContext & UserContext & Mocha.Context;
+
+const serviceName = "mongodb";
+const collectionName = "test-collection";
+
 describe.skipIf(environment.missingServer, "MongoDB Client", function () {
   this.timeout(60_000); // TODO: Temporarily hardcoded until envs are set up.
   importAppBefore(buildAppConfig("with-flx").anonAuth().flexibleSync());
+  authenticateUserBefore();
```

Comment:
If we add tests after the very last `it` test, it will probably be problematic to use this `authenticateUserBefore()` since it only logs in 1 time (`before`). I previously logged in the user `beforeEach` test because we're also logging out the user in the last test. So for future tests, the user will not be logged in if using `authenticateUserBefore()`. Since the previous login helper just returns `app.currentUser` if it's already logged in, I don't think there should be that much overhead having it.